### PR TITLE
fix: Change @pagination-font-family value

### LINKED ISSUE #26175: Change Pagination font family from Arial to default
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
The inconsistency between font families in the Pagination component and the rest of UI elements.

### What does the proposed API look like?
Change `@pagination-font-family` variable value in `default.less` to be `@font-family`.

### PR DESCRIPTION
**Pull request by albertms10**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#26175

### 💡 Background and solution

The inconsistency between font families in the Pagination component and the rest of UI elements.

### 📝 Changelog

Change `@pagination-font-family` variable value in `default.less` to `@font-family`.

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by xrkffgg — APPROVED** [review]

**Comment by afc163** [comment]
It affects ellipsis style.

image

**Comment by 07akioni** [comment]
You can add
```css
ant-pagination-item-ellipsis {
  font-family: Arial;
}
```

**Comment by afc163** [comment]
ping @albertms10

**Comment by albertms10** [comment]
Should I add a generic fallback font to pass the `font-family-no-missing-generic-family-keyword` check, just as `Arial, sans-serif`?

**Comment by albertms10** [comment]
Helvetica does also work well with these ellipsis.

**Comment by albertms10** [comment]
Updating tests… forgive me!

File: components/tree/__tests__/__snapshots__/directory.test.js.snap [hunk 1]
```diff
@@ -724,7 +724,6 @@ exports[`Directory Tree expand click 1`] = `
   </div>
   <div
     class="ant-tree-list"
-    disabled=""
```
Comment (afc163) [inline components/tree/__tests__/__snapshots__/directory.test.js.snap:727]: Please rebase master.

**Comment by afc163** [comment]
Mixed other changes.

**Comment by albertms10** [comment]
I forgot to create a branch when I forked the repo, that may be the reason… Shall I create a new clean Pull request?

**Pull request closed without merging**